include/solvers/euler2_solver.h

```diff
@@ -70,6 +70,21 @@ class Euler2Solver : public FirstOrderUnsteadySolver
    */
   virtual Real error_order() const override;
 
+  /**
+   * A method to integrate the system::QoI functionals.
+   */
+  virtual void integrate_qoi_timestep() override;
+
+  /**
+   * A method to compute the adjoint refinement error estimate at the current timestep.
+   * int_{tstep_start}^{tstep_end} R(u^h,z) dt
+   * Fills in an ErrorVector that contains the weighted sum of errors from all the QoIs and can be used to guide AMR.
+   * Also fills in a map that links QoI indices to spatially integrated error estimates for the QoI with that index.
+   * The trapezoidal rule (for temporally smooth QoI) or left Riemann sum (for an instantaneous QoI) is used
```

Comment:
If we're using the trapezoidal rule regardless of theta, that's inconsistent and we need to fix it or at the very least error out when we see theta!=0.5.

If we're weighting our integration consistently with theta then this comment should say so instead.

Reply:
The comment is outdated, will bring it up to speed with the more general implementation we have now.

Reply:
Done.